inductiva/localization/translations/en.json

```
@@ -8,8 +8,52 @@
         "Unable to check for available updates for the 'inductiva' package.",
         "The following error occurred:\n  {0}"
     ],
-    "kill-request-sent":["Successfully sent kill request for task {0}.",
+    "task-request-sent":["Successfully sent kill request for task {0}.",
```

`task-request-sent` (and `kill-request-sent`) do not give provide enough context on what the msg is used for. Perhaps (make sure it is used in the right place because I didn't see an update for it's usage):

```suggestion
    "task-kill-request-sent":["Successfully sent kill request for task {0}.",
```

---

I've changed the name of the variable and forgot to change the place where it was called. Fixed now.